python/py-fido2/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-fido2
+github.setup        Yubico python-fido2 0.3.0
+categories          python security crypto
+platforms           darwin
+license             BSD
+maintainers         {@amake madlon-kay.com:aaron+macports} openmaintainer
+
+description         Provides library functionality for communicating with a \
```

Comment:
I'd probably keep this as the long description but provide a shorter (one liner) for `description`.

Reply:
Thanks, done.